app/assets/stylesheets/global.scss

```diff
@@ -115,3 +115,21 @@ textarea {
   font-family: $body-fonts;
   padding: 0.7rem 1rem 0.7rem 1rem;
 }
+
+a.internal-link {
```

Comment:
Since internal link only works within lesson files can we move this class into the lesson content component style sheet please? 
https://github.com/TheOdinProject/theodinproject/blob/master/app/assets/stylesheets/components/lesson/lesson_content.scss

Reply:
I was thinking the same but it works here too : 
![internallink3](https://user-images.githubusercontent.com/44884803/52842628-89767880-30b4-11e9-9915-160f48da9b95.jpg)

Reply:
ah, very nice!

In that case I would probably put this in its own component style sheet.